pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/MultiStageAccountingTest.java

```diff
@@ -0,0 +1,246 @@
+/**
```

Comment:
do we want to add a test case of query being killed in MSE execution?

Reply:
Since the trheading model is different, I couldnt make those tests work. I'll give it a try again.